test/browsercontext.spec.js

```diff
@@ -121,6 +128,30 @@ describe('BrowserContext', function() {
     let error = await promise;
     expect(error.message).toContain('Context closed');
   });
+  it('should support viewport in persistent context', async({browserType, defaultBrowserOptions}) => {
```

Comment:
there are `defaultbrowsercontext.spec.js` tests that might be a better home for this.